Proton/Sources/Core/RichTextView.swift

```diff
@@ -22,15 +22,17 @@ import Foundation
 import UIKit
 
 class RichTextView: AutogrowingTextView {
-    private let storage = PRTextStorage()
+    
+    /// Equivilant, strongly-typed alternative to `textStorage`
```

Comment:
Maybe, `Equivalent`?
Also, as far as I can see `richTextStorage` is used only [in one place outside of this class](https://github.com/rajdeep/proton/blob/25225154f7236831389b6bfd58b5c8b31da260bf/Proton/Sources/Core/RichTextEditorContext.swift#L178) and probably can become private by introducing `defaultFont` getter right in `RichTextView` (wrapper on `PRTextStorage::defaultFont`)

Reply:
- [ ] private `richTextStorage`